spk/jellyfin/Makefile

```diff
@@ -14,7 +14,7 @@ MAINTAINER = stevenliuit
 DESCRIPTION = "The Free Software Media System. It is an alternative to the proprietary Emby and Plex."
 DISPLAY_NAME = Jellyfin
 STARTABLE = yes
-CHANGELOG = "1. Update jellyfin to 10.8.5 2.update dotnet 6.0.401"
+CHANGELOG = "1. Update jellyfin to 10.8.9 2.update dotnet 6.0.405"
```

Comment:
this should be updated to:

`CHANGELOG = "1. Update jellyfin to 10.8.9<br>2. Update dotnet 6.0.405"`

Reply:
done